src/player.cpp

```diff
@@ -3392,8 +3393,8 @@ int player::print_aim_bars( WINDOW *w, int line_number, item *weapon, Creature *
     // This simplifies the calculation greatly, that's intentional.
     const std::array<std::pair<double, char>, 3> ratings =
         {{ std::make_pair(0.1, '*'), std::make_pair(0.4, '+'), std::make_pair(0.6, '|') }};
-    const std::string confidence_label = _("Confidence: ");
```

Comment:
Oh, well I decided that the original colon was fine :)

it matches up with the "Symbols:" above
It means that those fields are uneffected for localisation.
There's still loads of space for the meter itself (29 characters vs 30).